### ISSUE TITLE
Increasing terminal width doesn't address truncation

### ISSUE THREAD

**Issue by denelon** [microsoft/winget-cli#3071]
> My terminal window _was_ wide enough:
> 
> ![image](https://user-images.githubusercontent.com/12860484/224758932-b0826738-9f9b-4f64-8d40-458628c9cb49.png)

Is there an existing issue for this?

_Originally posted by @sba923 in https://github.com/microsoft/winget-cli/issues/3069#issuecomment-1466524470_

**Comment by Trenly** [comment]
@denelon - I'm wondering if this is because it was piped.

@sba923 - does the same thing occur with `winget list 'ImageMagick`?

**Comment by sba923** [comment]
> @sba923 - does the same thing occur with `winget list 'ImageMagick`?

Nope it doesn't.

Seems indeed pipe-related.

![image](https://user-images.githubusercontent.com/12860484/224992360-5d1aabcf-8ea5-4d0c-8e8b-19e51ac90998.png)

**Comment by denelon** [comment]
That makes sense. We have a default value when we can't "get width" from the terminal.

**Comment by Trenly** [comment]
Looking at this more, I think there's also an error in how the console width is determined. For example, running `winget list` doesn't use the full console width on my machine, even when the buffer width is set to 5000. When running `winget list Microsoft` the table is larger, but there are still truncated entries -
```
Name                                                         Id                                                Version             Available
---------------------------------------------------------------------------------------------------------------------------------------------------
|
Microsoft Visual C++ 2008 Redistributable - x86 9.0.30729.6… Microsoft.VCRedist.2008.x86                       9.0.30729.6161
```

**Comment by Trenly** [comment]
[Policy] Area-Output

**Comment by dftf-stu** [comment]
Yes, for some packages it makes it impossible to see the available languages:

<img width="1366" height="738" alt="Image" src="https://github.com/user-attachments/assets/fd4fbee6-86e1-45bf-8719-12b1c6473065" />

Above: output from `winget search firefox`


Reducing the font size, or outputting to a file (`winget search firefox > c:\temp\output.txt`) changes nothing
